bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    # We have Rototrans_matrix_humerus_ulna, so, we are now able to transform the coordinates of any point in the ulna
</pre><p>Put this after the results as plain text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>